eng/targets/Settings.props

```diff
@@ -10,6 +10,12 @@
 
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
 
+    <!-- Workaround for old Microsoft.VisualStudio.Progression.* packages -->
+    <NoWarn>$(NoWarn);VSIXCompatibility1001</NoWarn>
```

Comment:
I think we can remove this now?

Reply:
No, newer Progression packages still not available: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/516234

Reply:
This is added to the globalconfig, right?

Reply:
It is and for some reason it does not suppress all warnings :(. I'll follow up.